README.md

```diff
@@ -1,2 +1,31 @@
-# nuodata-example
-Example for Nuodata.io
+## Synopsis
+This file takes an excel file and sends it to the specified URI database. In this case it's made for sending updates to a database on nuodata.io.
+
+## Pre-reqs
+run the following in the command line:
+```
+pip install -r requirements.txt
```

Comment:
test it in a clean python environment using virtualenv.

http://docs.python-guide.org/en/latest/dev/virtualenvs/